client/src/components/questions/QuestionSearch.jsx

```diff
@@ -1,11 +1,18 @@
 /* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
+import styled from 'styled-components';
 
-const QuestionSearch = () => {
+const StyledSearchBar = styled.input`
+  width: 100%;
+  height: 7vh;
+
+`;
+
+const QuestionSearch = ({ handleChange, value }) => {
```

Comment:
I would suggest using rem or as a unit of measurement, it doesn't seem to make a difference but it is best practice opposed to using vh and vw. Pretty easy transition too.
https://gist.github.com/basham/2175a16ab7c60ce8e001

Reply:
Thanks, I was wondering that for the search bar.